### ISSUE TITLE
cargo build fails on macOS 10.12

### ISSUE THREAD

**Issue by melchor629** [harababurel/gcsf#11]
While installing `gcsf` using the `cargo install gcsf` command, it fails with this error:
```
error[E0432]: unresolved import `libc::EREMOTEIO`
 --> src/gcsf/filesystem.rs:7:29
  |
7 | use libc::{ENOENT, ENOTDIR, EREMOTEIO};
  |                             ^^^^^^^^^ no `EREMOTEIO` in the root. Did you mean to use `EREMOTE`?
```

I have rustc 1.27.0 and cargo 1.26.0 (installed via `brew`) and osxfuse 3.8.0.

If it helps, I tried to look for this `EREMOTEIO` constant in [libc's documentation for macOS](https://rust-lang.github.io/libc/x86_64-apple-darwin/libc/) and it doesn't exist. But on [Linux it does](https://rust-lang.github.io/libc/x86_64-unknown-linux-gnu/libc/).

**Comment by harababurel** [comment]
I replaced `EREMOTEIO` with `EREMOTE` and published a new version (0.1.7).  Try installing it again and let me know if you still encounter this problem.

**Comment by melchor629** [comment]
Pretty fast. It worked, now compiles. Thanks :)
